doc/api/errors.md

```diff
@@ -563,6 +563,17 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes">
+## Node.js Error Codes
+
```

Comment:
We probably need a section explaining the new error system(like the section on "system errors" above mentioning those would have `error.code`, `error. errno` and `error. syscall` properties, then list the error codes). Going straight to error code descriptions seems pretty confusing if people read it from top to bottom...